Pod/Classes/SWSegmentedItem.swift

```diff
@@ -7,7 +7,6 @@
 
 import Foundation
 
-private let Padding: CGFloat = 5
```

Comment:
I removed this padding because I thought this was unnecessary as there is already an automatic padding for the label in UIButton.